### PR TITLE
[Fix] Reorganize Develop Section

### PR DESCRIPTION
**Pull request by albertov19**
This PR reorganizes a bit the Develop section by changing Blockchains to Parachains, moving Integrations to Toolkit and fix some minor things here and there.

Goes with https://github.com/papermoonio/polkadot-mkdocs/pull/43 to fix a home URL

### CODE REVIEW

File: tutorials/polkadot-sdk/parachains/connect-to-relay-chain/prepare-relay-chain.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ Before diving into this tutorial, it's recommended that you have a basic underst
 
 To complete this tutorial, ensure that you have:
 
-- Installed Rust and the Rust toolchain. Refer to the [Installation](/develop/blockchains/get-started/install-polkadot-sdk){target=\_blank} guide for step-by-step instructions on setting up your development environment
-- Completed [Launch a Local Solochain](/tutorials/polkadot-sdk/parachains/local-chain/launch-a-local-solochain/){target=\_blank} tutorial and know how to compile and run a Polkadot SDK-based node
+- Installed Rust and the Rust toolchain. Refer to the [Installation](/develop/parachains/get-started/install-polkadot-sdk){target=\_blank} guide for step-by-step instructions on setting up your development environment
+- Completed [Build a Local Blockchain](/tutorials/blockchains/custom-blockchains/build-a-blockchain/build-a-local-blockchain){target=\_blank} tutorial and know how to compile and run a Polkadot SDK-based node
```
Comment (nhussein11) [inline tutorials/polkadot-sdk/parachains/connect-to-relay-chain/prepare-relay-chain.md:26]:
```suggestion
- Completed [Launch a Local Solochain](/tutorials/polkadot-sdk/parachains/local-chain/launch-a-local-solochain/){target=\_blank} tutorial and know how to compile and run a Polkadot SDK-based node
```

File: develop/development-pathways.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ The Polkadot ecosystem provides multiple development pathways:
 graph TD
     A[Development Pathways]
     A --> B[Smart Contract Development]
-    A --> C[Blockchain Development]
+    A --> C[Parachain Development]
     A --> D[Client-side Development]
```
Comment (nhussein11) [inline develop/development-pathways.md:20]: Not sure about this one tbh, but just wanted to point it out:

```suggestion
    A --> D[Client-Side Development]
```

**Review by nhussein11 — COMMENTED** [review]
I re-read the complete structure, and I find it a bit odd that the term "parachains" is being widely used as a synonym for "blockchains." Perhaps I’m still tied to an older mindset, but if we are moving forward with this rephrasing and restructuring, we should include a **CLEAR clarification** somewhere, before jumping into the develop section, that explains the following:

> The widespread term right now is "parachains." Parachains are "parallel blockchains secured by Polkadot."

At this point, I find it harder to follow the narrative using "parachains" instead of "blockchains". It may stem from my previous understanding, where parachains were intrinsically linked to Cumulus and viewed as entirely distinct from solochains, idk

File: develop/toolkit/integrations/wallets.md [hunk 3]
```diff
@@ -18,19 +18,27 @@ Wallet types fall into two categories based on their connection to the internet:
...
+    A non-custodial, mobile-first wallet for managing assets and interacting with the Polkadot and Kusama ecosystems. It supports staking, governance, cross-chain transfers, and crowdloans. With advanced features, seamless multi-network support, and strong security, Nova Wallet empowers users to explore the full potential of Polkadot parachains on the go.
+
+    [:octicons-arrow-right-24: Reference](https://talisman.xyz/){target=\_blank}
+
 -   __Talisman__
 
     ---
 
-    With Talisman, you can securely store assets, manage your portfolio, and interact with Polkadot and Ethereum applications. It supports Web3 apps, asset storage, and account management across over 150 Polkadot SDK-based and EVM networks. Additional features include NFT management, Ledger support, fiat on-ramp, and portfolio tracking
+    a non-custodial web browser extension that allows you to manage your portfolio and interact with Polkadot and Ethereum applications. It supports Web3 apps, asset storage, and account management across over 150 Polkadot SDK-based and EVM networks. Features include NFT management, Ledger support, fiat on-ramp, and portfolio tracking.
```
Comment (eshaben) [inline develop/toolkit/integrations/wallets.md:33]:
```suggestion
    A non-custodial web browser extension that allows you to manage your portfolio and interact with Polkadot and Ethereum applications. It supports Web3 apps, asset storage, and account management across over 150 Polkadot SDK-based and EVM networks. Features include NFT management, Ledger support, fiat on-ramp, and portfolio tracking.
```

File: develop/toolkit/integrations/wallets.md [hunk 4]
```diff
@@ -18,19 +18,27 @@ Wallet types fall into two categories based on their connection to the internet:
...
 
     [:octicons-arrow-right-24: Reference](https://talisman.xyz/){target=\_blank}
 
 -  __Subwallet__
 
     ---
 
-    A non-custodial Polkadot & Ethereum wallet. Track, send, receive, and monitor multi-chain assets on 150+ networks. Import account with seed phrase, private key, QR code, and JSON file. Import token & NFT, attach read-only account. XCM Transfer, NFT Management, Parity Signer & Ledger support, light clients support, EVM DApp support, MetaMask compatibility, custom endpoints, fiat on-ramp, phishing detection, transaction history.
+    A non-custodial web browser extension and mobile wallet for Polkadot & Ethereum. Track, send, receive, and monitor multi-chain assets on 150+ networks. Import account with seed phrase, private key, QR code, and JSON file. Import token & NFT, attach read-only account. XCM Transfer, NFT Management, Parity Signer & Ledger support, light clients support, EVM DApp support, MetaMask compatibility, custom endpoints, fiat on-ramp, phishing detection, transaction history.
```
Comment (eshaben) [inline develop/toolkit/integrations/wallets.md:41]:
```suggestion
    A non-custodial web browser extension and mobile wallet for Polkadot and Ethereum. Track, send, receive, and monitor multi-chain assets on 150+ networks. Import account with seed phrase, private key, QR code, and JSON file. Import token & NFT, attach read-only account. XCM Transfer, NFT Management, Parity Signer & Ledger support, light clients support, EVM dApp support, MetaMask compatibility, custom endpoints, fiat on-ramp, phishing detection, transaction history.
```

**Review by eshaben — APPROVED** [review]
Should "Build Custom Blockchains" be "Build Custom Parachains"?

File: develop/parachains/customize-parachain/benchmarking.md [hunk 5]
```diff
@@ -53,19 +53,19 @@ The benchmarking tool runs multiple iterations to model worst-case execution tim
 Before writing benchmark tests, you need to ensure the `frame-benchmarking` crate is included in your pallet's `Cargo.toml` similar to the following:
 
 ```toml title="Cargo.toml"
---8<-- 'code/develop/blockchains/custom-blockchains/benchmarking/cargo.toml:1:1'
+--8<-- 'code/develop/parachains/customize-parachain/benchmarking/cargo.toml:1:1'
```
Comment (0xLucca) [inline develop/parachains/customize-parachain/benchmarking.md:56]: First line was rendering empty, this change fixes that
```suggestion
--8<-- 'code/develop/parachains/customize-parachain/benchmarking/cargo.toml::1'
```

**Review by 0xLucca — APPROVED** [review]
LGTM

**Pull request merged**